packages/blockchain-utils/index.ts

```diff
@@ -1,8 +1,12 @@
-import { Provider, Callback, JsonRPCResponse } from "web3/providers"
-import { parsedUriObject } from "typings"
+import { parsedUriObject } from "typings";
```

Comment:
Does this need to be here? Looks like it isn't used.

Reply:
https://github.com/trufflesuite/truffle/blob/edd533722a361872ba7a24f46b14878c1009358a/packages/blockchain-utils/index.ts#L40

Reply:
Oh my mistake, I only searched the diff.